src/c_kzg_4844.c

```diff
@@ -1089,12 +1089,12 @@ static C_KZG_RET hash_to_bytes(uint8_t out[32],
   if (bytes == NULL) return C_KZG_MALLOC;
 
   memcpy(bytes, FIAT_SHAMIR_PROTOCOL_DOMAIN, 16);
-  bytes_of_uint64(&bytes[16], n);
-  bytes_of_uint64(&bytes[16 + 8], FIELD_ELEMENTS_PER_BLOB);
+  bytes_of_uint64(&bytes[16], FIELD_ELEMENTS_PER_BLOB);
```

Comment:
See https://github.com/ethereum/consensus-specs/blob/dev/specs/eip4844/polynomial-commitments.md#hash_to_bls_field where we put the degree of the polynomial first and then the number of polynomials

Reply:
nice - I must have got confused by the order of the declarations not matching the order of concatenation in the spec.